### ISSUE TITLE
[Docs] Improve Building section

### ISSUE THREAD

**Issue by Drsheppard01** [mgord9518/shappimage#4]
I think you need to rewrite it to a step-by-step way of creating a ShImg

1. Create AppDir using this manual: https://github.com/AppImage/AppImageKit/wiki/AppDir
2. Compress AppDir ```mksquashfs ~/Appdir appname.sqfs -b1M -comp lz4 -keep-as-directory -info

**Comment by mgord9518** [comment]
Yeah I agree, the docs here were whipped up without much though initially, I'm currently going through and trying to make them easier to follow

**Comment by mgord9518** [comment]
I've made some changes, I'll keep this issue open for now though as it still needs to be further improved

**Comment by Drsheppard01** [comment]
When i concatenate runtime and squashfs and launch this file i has this output:

usage: /butterfly.shImg [-hfdxlov] <archive> [mountpoint] 
description: read SquashFS images

First of all, I wanna ask if I've done everything right?
Secondly: How to launch (mount and launch) shImg without TUI
Tertiary: Looks like need to update Building section after this improvements

**Comment by mgord9518** [comment]
Did you accidentally concatenate the squashfuse binary instead of the sh runtime? If not then there's probably a bug that I'll have to investigate. I'll mess around a bit on my days off and see

Without using the console it depends on your DE, and this is the most difficult part imo. GNOME is basically a no-go since you can't launch shell script directly, KDE works but it pops up a dialog asking if you want to edit it or run, which also isn't optimal since the shImg contains binary information and tend to be too large to open in an editor

And yeah, I'll probably end up writing a simple tool to build them then update based on that

**Comment by Drsheppard01** [comment]
i use this stroke from manual `cat runtime AppDir.sqfs > app.shImg` and launch this script in terminal (cause i use GNOME)

**Comment by mgord9518** [comment]
Oh I see, I believe `runtime` is actually a temporary file while building the actual runtime, which is labeled something like `runtime-x86_64-lz4`

That's my bad, I can't work from my computer currently but I'll mess around with it on my phone and see if I can remove some of the confusion in the build script itself then update the docs
